corehq/apps/app_manager/models.py

```diff
@@ -393,6 +402,25 @@ class LoadUpdateAction(AdvancedAction):
     auto_select = SchemaProperty(AutoSelectCase, default=None)
     show_product_stock = BooleanProperty(default=False)
     product_program = StringProperty()
+    case_index = SchemaProperty(CaseIndex)
```

Comment:
Thanks. I think that's probably the right thing to do, instead of checking the value of tag.